aws/aws-explanation/eks-snaps.rst

```diff
@@ -0,0 +1,26 @@
+Snap Usage in EKS Worker Images
+===============================
+
+The EKS worker images bundle several snaps that provide
+kubernetes components. This page will provide a brief
+explanation on what snaps are bundled and why.
+
+Snaps
+=====
+
+.. list-table::
+   :header-rows: 1
+
+   * - **Snap Name**
+     - **Provides**
+     - **Bundled Since**
+   * - kubelet-eks
+     - kubelet binary - primary configuration mechanism
```

Comment:
This is slightly confusing, can we expand the explanation a bit? Maybe something like: 

> the [kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/) binary, which includes the primary configuration mechanism for kubelets

Since this content is an explanation, we can give more details and be more verbose than say a how-to guide. So including appropriate external links would be a good idea here.